src/Assertion/AssertionContainerInterface.php

```diff
@@ -19,29 +19,9 @@
 
 declare(strict_types=1);
 
-namespace ZfcRbac\Role;
+namespace ZfcRbac\Assertion;
 
-use Zend\ServiceManager\AbstractPluginManager;
-use Zend\ServiceManager\Factory\InvokableFactory;
-use ZfcRbac\Container\ObjectRepositoryRoleProviderFactory;
-
-/**
- * Plugin manager to create role providers
- *
- * @method RoleProviderInterface get($name)
- *
- * @author  Michaël Gallego <mic.gallego@gmail.com>
- * @license MIT
- */
-final class RoleProviderPluginManager extends AbstractPluginManager
+interface AssertionContainerInterface
```

Comment:
What about extending PSR-11 `ContainerInterface` here?

Reply:
Not sure about this. What would we get from this? but yes maybe we can.
@prolic @basz what you think?

Reply:
why not. it's a container